api/v01/entities/vrp_result.rb

```diff
@@ -45,16 +46,18 @@ class VrpResultSolutionRouteActivityDetails < Grape::Entity
     end
 
     class VRPResultDetailedCosts < Grape::Entity
-      expose :total, documentation: { type: String, desc: 'Cumulated cost' }
-      expose :fixed, documentation: { type: String, desc: 'Cost associated to the use of the vehicle' }
-      expose :time, documentation: { type: String, desc: 'Cost associated to the time dimension' }
-      expose :distance, documentation: { type: String, desc: 'Cost associated to the distance dimension' }
-      expose :value, documentation: { type: String, desc: 'Cost associated to the value dimension' }
-      expose :lateness, documentation: { type: String, desc: 'Cost associated to late arrival' }
-      expose :overload, documentation: { type: String, desc: 'Cost associated to quantities overload' }
+      expose :total, documentation: { type: Float, desc: 'Cumulated cost' }
+      expose :fixed, documentation: { type: Float, desc: 'Cost associated to the use of the vehicle' }
+      expose :time, documentation: { type: Float, desc: 'Cost associated to the time dimension' }
+      expose :distance, documentation: { type: Float, desc: 'Cost associated to the distance dimension' }
+      expose :value, documentation: { type: Float, desc: 'Cost associated to the value dimension' }
+      expose :lateness, documentation: { type: Float, desc: 'Cost associated to late arrival' }
+      expose :overload, documentation: { type: Float, desc: 'Cost associated to quantities overload' }
     end
 
     class VrpResultSolutionRouteActivities < Grape::Entity
+      expose :day_week_num, documentation: { type: String, desc: '' }
+      expose :day_week, documentation: { type: String, desc: '' }
```

Comment:
Do we need both `day_week_num` and `day_week`?

Reply:
I think we do. They are both needed in CSV file to import in mapotempo. Today initial generates this CSV file from JSON file returned by API (which contains those fields).